provision-contest/ansible/roles/domlogo/tasks/main.yml

```diff
@@ -44,10 +44,13 @@
     owner: domjudge
     group: domjudge
 
-- name: Add autostart shortcuts
+- name: Copy domlogo systemd unit file
   template:
-    src: "domjudgelogo.desktop.j2"
-    dest: "/home/domjudge/.config/autostart/domjudgelogo.desktop"
-    owner: domjudge
-    group: domjudge
-    mode: 0755
+    src: "domlogo.service.j2"
+    dest: "/etc/systemd/system/domlogo.service"
+
+- name: Enable and start domlogo
+  systemd:
+    name: domlogo
+    enabled: true
+    state: started
```

Comment:
```suggestion
    state: started
```

Reply:
I don't see what is different here, github diff UI is failing me

Reply:
It adds the EOL.

Reply:
GitHub says the suggestion is invalid since no changes were made to the code :-D

Reply:
Strange, in the past it would add the `EOL`